xc7/primitives/slicem/Ndram/d_dram.pb_type.xml

```diff
@@ -75,7 +78,34 @@
      <direct name="WE_L" input="D_DRAM.WE" output="DPRAM32_O5.WE" />
 
      <direct name="O5" input="DPRAM32_O5.O" output="D_DRAM.O5" />
-     <direct name="SO5" input="DPRAM32_O5.O" output="D_DRAM.SO5_32" />
+     <direct name="SO5" input="DPRAM32_O5.O" output="D_DRAM.SO5_32" >
+      <pack_pattern  out_port="D_DRAM.SO5_32" name="DRAM_8O_32" in_port="DPRAM32_O5.O" />
+      <pack_pattern  out_port="D_DRAM.SO5_32" name="DRAM_DP_32_HI" in_port="DPRAM32_O5.O" />
+     </direct>
+   </interconnect>
+  </mode>
+ <mode name="32_SINGLE_PORT">
+   <pb_type name="SPRAM32_O6" num_pb="1" blif_model=".subckt SPRAM32">
+    <xi:include href="spram32.pb_type.xml" xpointer="xpointer(pb_type/child::node()[local-name()!='metadata'])" />
+    <metadata>
+     <xi:include href="spram32.pb_type.xml" xpointer="xpointer(pb_type/metadata/child::node())" />
+     <meta name="fasm_params">
+      INIT[63:32] = INIT_00
+      INIT[31:0] = INIT_ZERO
```

Comment:
Can you add an XML comment explaining that this is known to be wrong?  Open a new issue about fixing the RAM32's in the future.